rc/lsp.kak

```diff
@@ -105,6 +105,25 @@ declare-option -hidden range-specs rust_analyzer_inlay_hints
 declare-option -hidden range-specs lsp_diagnostics
 declare-option -hidden str lsp_project_root
 
+# Faces used for Markdown highlighting in the info box
+# We can't assume the user's theme and any best effort attempts
+# for picking colors will likely not work for many users.
+# Therefore, by default, most are not defined until the user does so
+# explicitly.
+# set-face global InfoDefault Information
+# set-face global InfoHeader Information
+# set-face global InfoBlock Information
+# set-face global InfoBullet Information
+# set-face global InfoLink Information
+# set-face global InfoMono Information
+# set-face global InfoLinkMono Information
+# set-face global InfoRule Information
+# set-face global InfoDiagnosticHint Information
+# set-face global InfoDiagnosticInformation Information
+# set-face global InfoDiagnosticWarning Information
+set-face global InfoDiagnosticError Error
```

Comment:
I'm not quite sure about the names yet.

Keeping it generic allows reuse with other plugins that might want to render markup in the info box, and reduces the number of faces a user/scheme creator needs to maintain.
But it also limits customizability and prevents giving kak-lsp's output different colours than those other plugins.

Reply:
Hmm our other faces are not namespaced either. Most other plugins probably define few faces (?).
So if we are the first to define a bunch of info faces, other plugins can decide to either use ours, or define theirs if they want more customizability. Or is there already some plugin that has colors for info markup?
I personally never customize any colors, so I'm not an expert but fewer colors seem nice.

---

In theory, we could expose two faces with inheritance like `set-face global LSPInfoDefault InfoDefault` where `InfoDefault` is the generic one. Then it's enough if colorschemes define the generic one.
Since we can always add the namespaced names later, maybe let's stick to the generic names.
Though maybe it's better to move to namespaced names in general... I'm not sure